### PR TITLE
feat(Comment): add component

### LINKED ISSUE #187: <Comment /> Component
http://semantic-ui.com/views/comment.html

### PR DESCRIPTION
**Pull request by eanplatter**
WIP

Fixes #187

### CODE REVIEW

File: docs/app/Examples/views/Comment/Types/CommentCommentsExample.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        <Comment.Meta>
+          <span class="date">5 days ago</span>
+        </Comment.Meta>
+        <Comment.Text>
+          Dude, this is awesome. Thanks so much
+        </Commment.Text>
+        <Comment.Actions>
+          <a class="reply">Reply</a>
+        </Comment.Actions>
+      </Comment.Conent>
```
Comment (typehorror) [inline docs/app/Examples/views/Comment/Types/CommentCommentsExample.js:75]:
``` jsx
</Comment.Content>
```

**Comment by levithomason** [comment]
This branch needs updated to keep from getting to far behind the latest updates.

**Comment by layershifter** [comment]
Looks like dead PR, I can start it from scratch ⚡

**Comment by layershifter** [comment]
Replaced with #584.

**Comment by levithomason** [comment]
@eanplatter !!!!!!!!!!!!!!!!!!

**Pull request closed without merging**